GNUmakefile

```diff
@@ -51,13 +54,18 @@ endif
 
 INSTALL_NAME_TOOL := install_name_tool
 
-STORE_DIR        := store
-BINDIST_BASE_DIR := out/bindist
+STORE_DIR        := store/$(ARTIFACT)
+BINDIST_BASE_DIR := out/bindist/$(ARTIFACT)
 BINDIST_OUT_DIR  := $(BINDIST_BASE_DIR)/haskell-language-server-$(HLS_VERSION)
 
-CABAL_ARGS         ?= --store-dir=$(ROOT_DIR)/$(STORE_DIR)
-CABAL_INSTALL_ARGS ?= --disable-tests --disable-profiling -O2 --overwrite-policy=always --install-method=copy
-CABAL_INSTALL      := $(CABAL) $(CABAL_ARGS) v2-install
+CABAL_BASE_ARGS         ?= --store-dir=$(ROOT_DIR)/$(STORE_DIR)
+CABAL_ARGS              ?= --disable-tests --disable-profiling -O2
```

Comment:
Again, I'd rather these be clearly set earlier, it's very easy to miss them hidden down in here.

Reply:
Hidden? This is the main Makefile for the release bindidts. It belongs here, so they can be built outside of a CI as well.